pkg/gameserversets/controller_test.go

```diff
@@ -380,16 +380,16 @@ func TestComputeStatus(t *testing.T) {
 			AllocatedReplicas: 1,
 			Counters: map[string]agonesv1.AggregatedCounterStatus{
 				"firstCounter": {
-					AllocatedCount:    5,
-					AllocatedCapacity: 10,
-					Count:             50,
-					Capacity:          85,
+					AllocatedCount:    agonesv1.SafeAdd(5, 0),
```

Comment:
For these we'll want to leave as plain int64. 

You can add another Counter under multiple game servers similar to:
```
		gs1.Status.Counters = map[string]agonesv1.CounterStatus{
			"firstCounter":  {Count: 5, Capacity: 10},
			"secondCounter": {Count: 100, Capacity: 1000},
			"fullCounter":   {Count: 9223372036854775807, Capacity: 9223372036854775807},
		}
```

And under expected add:

```
				"fullCounter": {
					AllocatedCount:    9223372036854775807,
					AllocatedCapacity: 9223372036854775807,
					Count:             9223372036854775807,
					Capacity:          9223372036854775807,
				},
```